ui/@types/lichess/index.d.ts

```diff
@@ -161,8 +161,26 @@ interface Fipr {
   x64hash128(input: string, seed: number): string;
 }
 
+interface Api {
```

Comment:
This needs to be updated, I think

Reply:
socket is missing.

Reply:
the types only need to contain what lila itself uses.